File: spologp_v/lista2casa-C/listas 2/lista2sala/listas2casa/lista2casac/questao5.c
#include<stdio.h>
#include<math.h>

int main()
{
    float salariob, salariol, horast, desconto1, desconto2, desconto3, desconto4, adicional;
    printf("Digite o salário bruto do funcionário (em reais):");
    scanf("%f", &salariob);
    printf("Digite as horas trabalhadas:");
    scanf("%f", &horast);

if(salariob<800){
    printf("Não é necessário realizar nenhum desconto");
}

if(800<salariob<1600){
    desconto1 = (salariob*8)/100;
    desconto2 = (salariob*5)/100;
    salariol = salariob-desconto1-desconto2;
    //salariol = salariol-desconto2;
    printf("O salário líquido do funcionário é: %.2f\n", salariol);
}

else if(1600<salariob){
    desconto3 = (salariob*15)/100;
    desconto4 = (salariob*7)/100;
    salariol = salariob-desconto4-desconto3;
    printf("O salário líquido do funcionário é: R$ %.2f\n", salariol);
}

else if(horast>160){
    adicional = (horast*5)/100;
    salariol = salariol+adicional;
    printf("O funcionário não precisa de adicionais de horas extras R$ %.2f\n", salariol);
}

else{
    printf("Esse funcionário não precisa de adicionais de horas extras");
}

return 0;
}

File: spologp_v/lista2casa-python/lista2casa-python/questao5.py
salariob = float(input("Digite o salário bruto (em reais):"))
horast = float(input("Digite a quantidade de horas trabalhadas no mês:"))

if(salariob<800):
    print("Não serão realizados descontos")
elif(800.00<=salariob<=1600.00):
    salario_liquido = float
    desconto1 = (salariob*8)/100
    desconto2 = (salariob*5)/100
    salario_liquido = salariob-desconto1-desconto2
    print("O salário líquido é:", "R$", '%.2f' %salario_liquido)
elif(salariob>1600.00):
    salario_liquido = float
    desconto3 = (salariob*15)/100
    desconto4 = (salariob*7)/100
    salario_liquido = salariob-desconto4-desconto3
    print("O salário líquido é:", "R$", '%.2f' %salario_liquido)

if(horast>160):
    adicional = (horast*5)/100
    salario_liquido = salario_liquido+adicional
    print("O funcionário não precisa de adicionais de horas extras R$ %.2f\n", salario_liquido)
else:
    print("O salário não precisa de adicionais de horas extras")